examples/utils/include/traccc/examples/utils/printable.hpp

```diff
@@ -49,31 +53,32 @@ class configuration_list final : public configuration_printable {
     public:
     configuration_list();
 
-    void add_child(std::unique_ptr<configuration_printable>&& elem);
+    void add_child(std::unique_ptr<configuration_printable> elem);
 
-    void print_impl(std::string self_prefix, std::string child_prefix,
-                    std::size_t prefix_len,
-                    std::size_t max_key_width) const final;
+    void print_impl(const std::string& self_prefix,
+                    const std::string& child_prefix, std::size_t prefix_len,
+                    std::size_t max_key_width,
+                    std::ostream& out) const override;
 
-    std::size_t get_max_key_width_impl() const final;
+    std::size_t get_max_key_width_impl() const override;
```

Comment:
Is there a reason to replace all these `final` clauses with `override`?

Reply:
I wanted to reduce the number of SonarCloud reports. (Now that VSCode underlines these things for me...) Since SonarCloud insists that with the classes being `final` already, using `final` in these places is "too much".

I don't have a strong opinion about this, but also don't disagree with SonarCloud. In general I try to use `final` very sparingly. Didn't really find a situation yet where using `final` would've made a piece of code noticeably easier to maintain.

Reply:
Okay fair!